### ISSUE TITLE
NTR: Maternal folate deficiency (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7847]
New term request Maternal folate deficiency (elthat_160910034247): Folic acid; in utero; pregnancy; embryopathy; maternal; mother; prenatal; prenatally

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000165
- label: Maternal folate deficiency
- parents: [Pregnancy history (HP:0002686)](https://hpo.jax.org/app/browse/term/HP:0002686)
- pmid: PMID:31093652
- def: The mother of a fetus or index patient has a history of folate deficiency during the corresponding pregnancy.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7847
- synonyms:
